package.json

```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "mime": "^2.4.4",
     "wbn": "0.0.8",
+    "wbn-sign": "^0.0.1",
```

Comment:
Should this version be a bit stricter (i.e., `0.0.1` instead of `^0.0.1`), given that each new release of `wbn-sign` could introduce breaking changes until we release version `1.0.0` of `wbn-sign`?